src/main/java/com/social/accesstoken/services/OkAccessToken.java

```diff
@@ -25,8 +27,14 @@ public OkAccessToken(String url) {
 	public String getAccessToken(SocialNetwork socialNetwork){
 		if(isAuthCorrect(socialNetwork)){
 			log.debug("Login and pass is correct");
-			HtmlPage permissionPage = handleAutorizePage(autorizeEntity, socialNetwork);
-			HtmlPage accessTokenPage = handlePermissionPage(permissionPage);
+			
+			AuthorizationService authorizationService = new AuthorizationService();
+			HtmlPage permissionPage = authorizationService.handleAutorizePage(autorizeEntity, socialNetwork);
+			
+			PermissionService permissionService = new PermissionService(
+					AutorizeDictionary.OK_FORM_ELEMENT_PERMISSION, 
+					AutorizeDictionary.OK_BUTTON_NAME);
+			HtmlPage accessTokenPage = permissionService.handlePermissionPage(permissionPage);
```

Comment:
The same here it is not so clear why you are creating two same objects call two differents methods from the same class. Could you please explain it